src/compiler/crystal/command/clear_cache.cr

```diff
@@ -0,0 +1,22 @@
+# Implementation of the `crystal clear_cache` command
+
+class Crystal::Command
+  private def clear_cache
+    OptionParser.parse(@options) do |opts|
+      opts.banner = <<-'BANNER'
+        Usage: crystal clear_cache
+
+        Clears the compiler cache
+
+        Options:
+        BANNER
+
+      opts.on("-h", "--help", "Show this message") do
+        puts opts
+        exit
+      end
+    end
+    puts "clearing compiler cache at \"#{CacheDir.instance.dir}\""
```

Comment:
I think this command shouldn't need any output. Maybe make this conditional on the `--verbose` flag?

Reply:
added the verbose flag.